lib/Common/Util.cpp

```diff
@@ -21,7 +21,7 @@
 #include <boost/filesystem.hpp>
 #include <boost/algorithm/string.hpp>
 #include <Common/Util.h>
-#include <../src/../src/config/CryptoNoteConfig.h>
+#include <../src/config/CryptoNoteConfig.h>
```

Comment:
Can be simplified as `#include <CryptoNoteConfig.h>`

Reply:
And if this won't work then we should consider added `src` folder to includes in cmake configuration.

Reply:
simplified `#include <config/CryptoNoteConfig.h>` should be the best case i guess

Reply:
I think we must include `src` to the cmake configuration.

> fatal error C1083: Cannot open include file: 'config/CryptoNoteConfig.h': No such file or directory

Reply:
OK, in this case I think that it's safe to skip this update for this PR. I'll add `src` folder to includes in cmake and fix include C++ files in a separate PR.

Reply:
Ok, will will leave the includes as it is and we will fix / optimize it later.
What do you think of the general idea with the files in a separate folder? Makes sense or just gimmick?